egs2/chime4/enh1/run.sh

```diff
@@ -27,7 +27,7 @@ test_sets="et05_simu_isolated_1ch_track"
     --spk_num 1 \
     --ref_channel 4 \
     --local_data_opts "--extra-annotations ${extra_annotations} --stage 1 --stop-stage 2" \
-    --enh_config conf/tuning/train_enh_conv_tasnet.yaml \
+    --enh_config conf/tuning/train_enh_iva.yaml \
```

Comment:
Better to keep `conf/tuning/train_enh_conv_tasnet.yaml` as the default configuration. Or do you observe better performance with IVA? In that case, we can use it as default.

Reply:
BTW, could you also add the result of the newly added model to [egs2/chime4/enh1/README.md](https://github.com/espnet/espnet/blob/master/egs2/chime4/enh1/README.md), if they are available?